### PR TITLE
Format typescript; added repo and fixed dev option in package.json

### PR DESCRIPTION
**Pull request by savage13**
- Formatted typescript files
- Added 'repository' to package.json
- Fixed option to 'npm run dev' from --transpileOnly to --transpile-only

---
This change is [Reviewable](https://reviewable.io/reviews/zeldamods/radar/8)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,12 @@
   "description": "",
   "scripts": {
     "tsc": "tsc",
-    "dev": "ts-node-dev --respawn --transpileOnly ./app/app.ts",
     "prod": "node ./app/app.js"
```
Comment (leoetlino) [inline package.json:7]: missing comma at the end

File: package.json [hunk 2]
```diff
@@ -4,11 +4,12 @@
...
+    "dev": "ts-node-dev --respawn --transpile-only ./app/app.ts",
```
Comment (leoetlino) [inline package.json:8]: the trailing comma should be removed (not sure if npm can handle it)

**Comment by savage13** [comment]
Fixed the trailing comma.  If the formatting is ok, I can format the objmap repo the same way; at least the Typescript files.

**Comment by leoetlino** [comment]
Sure, that sounds good to me

**Review by leoetlino — APPROVED** [review]

**Pull request merged**